keps/sig-apps/3939-consider-terminating-pods-jobs/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: Terminating Pods Should be Included as Active Pods
+kep-number: 3939
+authors:
+  - "@kannon92"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+status: provisional
+creation-date: 2023-04-03
+reviewers:
```

Comment:
@alculquicondor or @mimowo what do I put in reviewers or approvers section?

Is this updated after a review?

Reply:
you can update as you go.
Reviewers are people who gave meaningful reviews to the KEP. Approvers are the ones who actually have the power to approve the KEP. In SIG Apps, that's soltysh.